### DISCUSSION TITLE
Offer option to choose multiple file locations for the File Field "Allowed Directory" parameter

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#2926] [Feature Requests]
Currently in EE6 it is possible to select an "Allowed Directory" parameter for a File Field when it is being defined in the CP.  The options offered comprise either "All" or any **one** of a list of all the File Locations defined in the EE6 instance.

It would be useful in some circumstances to offer the choice of a selection of file locations - as a half-way house between "All" and "one".  

In UI terms could be achieved by changing the radio-button selectors currently used for the "not All" choices to a checkbox group.  In terms of back-end the same layout / options offered if the "All" option is selected could be used, with the list of options in the "Directory" selector being constrained to just the file locations chosen in the checkbox group.

**Comment by PDXMan1** [comment]
When I have to leave it set to "All" then I can pretty much count on image files showing up in document folders. I would love to see this enhancement in EE. My clients would benefit from it often. Thank you for reviewing it!

**Comment by jcogs-design** [comment]
Bump!

**Comment by nep** [comment]
This crossed my mind again for the 99th time today. Any news? Seems like it's not too big a lift.  Is anyone in the community open to building a PR for it?
